src/store/ChartState.js

```diff
@@ -23,7 +23,7 @@ class ChartState {
         this.stxx.addEventListener('drawing', this.saveDrawings.bind(this));
     };
 
-    @action.bound updateProps({ id, settings, isConnectionOpened, symbol, granularity, chartType, startEpoch, endEpoch }) {
+    @action.bound updateProps({ id, settings, isConnectionOpened, symbol, granularity, chartType, startEpoch, endEpoch, supportsAnimation }) {
```

Comment:
For consistency with the rest of the API, we should call it `isAnimationEnabled`

we can set default value of in this line:  `...endEpoch, isAnimationEnabled = true }`